autoreject/tests/test_autoreject.py

```diff
@@ -364,3 +364,60 @@ def test_fnirs():
     assert reject["hbo"] < 0.001  # This is a very high value as sanity check
     assert reject["hbr"] < 0.001
     assert reject["hbr"] > 0.0
+
+
+def test_ecog():
```

Comment:
we follow PEP 257. See how docstrings should be formatted: https://peps.python.org/pep-0257/

Reply:
not sure to which place you're referring to, but I changed something, see in the next commit

Reply:
see: "Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description."

Reply:
ok thanks, pushed a new version